image/src/utils/util.js

```diff
@@ -0,0 +1,15 @@
+const { NESTED_RESPONSE, CUSTOM_FIELDS } = require('../config/config')
```

Comment:
If this line is not used, please remove it

Reply:
removed.